src/features/organizations/hooks/useUpcomingOrgEvents.ts

```diff
@@ -20,3 +20,17 @@ export default function useUpcomingOrgEvents(orgId: number): ZetkinEvent[] {
       ),
   });
 }
+
+export function useOrgEvents(orgId: number): ZetkinEvent[] {
+  const apiClient = useApiClient();
+  const list = useAppSelector(
+    (state) => state.organizations.eventsByOrgId[orgId]
+  );
+
+  return useRemoteList(list, {
+    actionOnLoad: () => orgEventsLoad(orgId),
+    actionOnSuccess: (data) => orgEventsLoaded([orgId, data]),
+    loader: () =>
+      apiClient.get<ZetkinEvent[]>(`/api/orgs/${orgId}/actions?recursive`),
```

Comment:
If the user has significant permissions, this will load ALL events (past and future) for the organization. For an organization that has been using Zetkin for a long time, and a user with admin permissions, this could tens of thousands of events.

Use the `filter=start_time…` query paramaters from the hook above.